src/Resources/skeleton/authenticator/login_form.tpl.php

```diff
@@ -11,7 +11,7 @@
 <?php if ($logout_setup): ?>
     {% if app.user %}
         <div class="mb-3">
-            You are logged in as {{ app.user.username }}, <a href="{{ path('app_logout') }}">Logout</a>
+            You are logged in as {{ app.user.userIdentifier }}, <a href="{{ path('app_logout') }}">Logout</a>
```

Comment:
I think this needs to be conditional, right? Based on what version of Symfony the code is being generated for? 

Looks like a legit bug though :)